modules/expo-bluesky-swiss-army/ios/ExpoBlueskySwissArmy.podspec

```diff
@@ -0,0 +1,21 @@
+Pod::Spec.new do |s|
+  s.name           = 'ExpoBlueskySwissArmy'
+  s.version        = '1.0.0'
+  s.summary        = 'A collection of native tools for Bluesky'
+  s.description    = 'A collection of native tools for Bluesky'
```

Comment:
Yea, we keep adding little modules for small things. Like ideally we can just throw this Google Play referrer stuff into this toolkit. I'm just holding off because there's a whole PR for referrers https://github.com/bluesky-social/social-app/pull/4514 that I'll just put into this module once we merge. Trying to simplify that down a bit.